source/hwPortUtils.py

```diff
@@ -70,7 +71,8 @@ class DEVPROPKEY(ctypes.Structure):
 
 class dummy(ctypes.Structure):
 	_fields_ = (("d1", DWORD), ("d2", WCHAR))
-	_pack_ = 1
+	# SetupAPI.h in the Windows headers includes pshpack8.h when 64 bit, pshpack1.h otherwise
+	_pack_ = 8 if platform.architecture()[0].startswith("64") else 1
```

Comment:
The [documentation for platform.architecture](https://docs.python.org/3/library/platform.html#platform.architecture) says:

> To get at the “64-bitness” of the current interpreter, it is more reliable to query the `sys.maxsize attribute`.

```suggestion
	_pack_ = 8 if sys.maxsize > 2**32 else 1
```